### PR TITLE
Update EVM.cs pagination

### PR DESCRIPTION
**Pull request by KBryan**
Updated EVM.cs for pagination.

### CODE REVIEW

**Comment by leon-do** [comment]
will need to update the prefab scripts

https://github.com/ChainSafe/web3.unity/blob/173f4acb0e9ace479e6522666e164e79a0106188/Web3Unity/Scripts/Prefabs/EVM/AllErc1155Example.cs#L22

https://github.com/ChainSafe/web3.unity/blob/173f4acb0e9ace479e6522666e164e79a0106188/Web3Unity/Scripts/Prefabs/EVM/AllErc721Example.cs#L22

```
string chain = "ethereum";
string network = "rinkeby"; // mainnet ropsten kovan rinkeby goerli
string account = "0xebc0e6232fb9d494060acf580105108444f7c696";
string contract = "";
int first = 500;
int skip = 0;
string response = await EVM.AllErc721(chain, network, account, contract);
```

**Comment by KBryan** [comment]
got it, updated.

**Review by leon-do — APPROVED** [review]

File: Web3Unity/Scripts/Prefabs/EVM/AllErc1155Example.cs [hunk 1]
```diff
@@ -19,6 +19,8 @@ async void Start()
         string network = "rinkeby"; // mainnet ropsten kovan rinkeby goerli
         string account = "0xebc0e6232fb9d494060acf580105108444f7c696";
         string contract = "";
+        int first = 500;
+        int skip = 0;
         string response = await EVM.AllErc1155(chain, network, account, contract);
```
Comment (leon-do) [inline Web3Unity/Scripts/Prefabs/EVM/AllErc1155Example.cs:24]: @KBryan you need to add `first` and `skip` into the params

`string response = await EVM.AllErc1155(chain, network, account, contract, first, skip);`

File: Web3Unity/Scripts/Prefabs/EVM/AllErc721Example.cs [hunk 2]
```diff
@@ -19,6 +19,8 @@ async void Start()
         string network = "rinkeby"; // mainnet ropsten kovan rinkeby goerli
         string account = "0xebc0e6232fb9d494060acf580105108444f7c696";
         string contract = "";
+        int first = 500;
+        int skip = 0;
         string response = await EVM.AllErc721(chain, network, account, contract);
```
Comment (leon-do) [inline Web3Unity/Scripts/Prefabs/EVM/AllErc721Example.cs:24]: @KBryan you need to add `first` and `skip` into the params

`string response = await EVM.AllErc1155(chain, network, account, contract, first, skip);`

**Review by leon-do — APPROVED** [review]

**Pull request merged**